src/main/java/uk/ac/ebi/ena/webin/cli/context/sequence/SequenceManifestReader.java

```diff
@@ -119,6 +126,9 @@ public SequenceManifestReader(
 		manifest.setName(getManifestReaderResult().getValue( Field.NAME ));
 		manifest.setDescription(getManifestReaderResult().getValue( Field.DESCRIPTION ));
 
+        manifest.setSubmissionTool(getManifestReaderResult().getValue(ReadsManifestReader.Field.SUBMISSION_TOOL));
+        manifest.setSubmissionToolVersion(getManifestReaderResult().getValue(ReadsManifestReader.Field.SUBMISSION_TOOL_VERSION));
```

Comment:
Should not point to ReadsManifestReader Field/Description. Should use the fields from this class. Ot we could add them into a common class.